.github/PULL_REQUEST_TEMPLATE/docs.md

```diff
@@ -0,0 +1,37 @@
+## Description
+
+Closes: #XXXX
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct `docs:` prefix in the PR title
+- [ ] targeted the correct branch (see [PR Targeting](https://github.com/cosmos/interchain-security/blob/main/CONTRIBUTING.md#pr-targeting))
+- [ ] provided a link to the relevant issue or specification
+- [ ] reviewed "Files changed" and left comments if necessary <!-- relevant if the changes are not obvious  -->
+- [ ] confirmed all CI checks have passed
+
+### Reviewers Checklist
+
+*All items are required. Please add a note if the item is not applicable and please add
+your handle next to the items reviewed if you only reviewed selected items.*
+
+I have...
+
+- [ ] Confirmed the correct `docs:` prefix in the PR title
+- [ ] Confirmed all author checklist items have been addressed 
+- [ ] Confirmed that this PR only changes documentation
+- [ ] Reviewed content for consistency
```

Comment:
Do you mean formatting consistency?
```suggestion
- [ ] Reviewed content for formatting consistency
```

Reply:
No. Content consistency. Changes throughout the docs should be consistent.